code/modules/mob/living/ventcrawling.dm

```diff
@@ -63,21 +63,21 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/components/unary
 		src << "<span class='warning'>This ventilation duct is not connected to anything!</span>"
 
 
-/mob/living/proc/add_ventcrawl(obj/machinery/atmospherics/components/starting_machine)
+/mob/living/proc/add_ventcrawl(obj/machinery/atmospherics/components/unary/starting_machine)
 	if(!istype(starting_machine) || !starting_machine.returnPipenet())
 		return
 	var/list/totalMembers = list()
-	var/datum/pipeline/starting_machine_parent = starting_machine.parents["p1"]
-	totalMembers |= starting_machine_parent.members
-	totalMembers |= starting_machine_parent.other_atmosmch
+	var/datum/pipeline/starting_machine_parent = starting_machine.parents[PARENT1]
+	totalMembers += starting_machine_parent.members
+	totalMembers += starting_machine_parent.other_atmosmch
 
 	for(var/obj/machinery/atmospherics/A in totalMembers)
 		if(!A.pipe_vision_img)
 			A.pipe_vision_img = image(A, A.loc, layer = 20, dir = A.dir)
 			//20 for being above darkness
-		pipes_shown |= A.pipe_vision_img
+		pipes_shown += A.pipe_vision_img
 		if(client)
-			client.images |= A.pipe_vision_img
+			client.images += A.pipe_vision_img
```

Comment:
discussion on #coderbus indicated that |= is slower than +=; because there's no way for there to be duplicate entries (dismissing varedit nonsense) |= is unnecessary